smaker/runner.py

```diff
@@ -4,10 +4,22 @@
 import os
 import snakemake
 from smaker import path_gen
+from tqdm import tqdm
+import copy
+import time
 
 def pretty_dump(blob):
     return json.dumps(blob, indent=4, sort_keys=True)
 
+class TqdmExtraFormat(tqdm):
+    """Provides a `total_time` format parameter"""
```

Comment:
I'd recommend using a standard docstring format such as the example [here](https://www.datacamp.com/community/tutorials/docstrings-python#fifth-sub)

And are the backticks meant to be used in markdown somehow?

Reply:
I copy pasted that from tqdm's documentation as a style change, didn't notice the docstring. But yeah, that's what sphinx is supposed to do, anyway. I'm hesitant to overly formalize any comment just because of the nature of the copy/paste style change hack, I'll maybe just add that it was taken from tqdm's documentation.

Reply:
sounds good.  At some point we should decided on doc standards.